### DISCUSSION TITLE
Is there console in pybricks.com???

### DISCUSSION THREAD

**Discussion by ben7147** [pybricks/support#1093] [General]
Because I want to use pip install keyboard.

**Comment by dlech** [comment]
What are you trying to accomplish? Pybricks uses MicroPython, so Python packages can't be used.

**Comment by laurensvalk** [comment]
Here’s how you can use the keyboard in your code. No additional packages required: https://pybricks.com/projects/tutorials/wireless/hub-to-device/pc-keyboard/
